### DISCUSSION TITLE
[Feature] Remove (or make configurable) or include pagination or utilize the main view (but filtered)

### DISCUSSION THREAD

**Discussion by bcsteeve** [immich-app/immich#3446] [Feature Request]
### The feature

I just went through the trouble of merging faces and found there's a limit of 1000 when viewing a tagged person.

Should either have no limit, or else allow the user to configure the limit, or else (possibly best) paginate the results.

Alternatively, recycle the main page's view with the dates on the right but filter to only include that tagged person (I recall someone saying they were working on this for albums, so maybe an overall filtered view solves albums as well as people/places).

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by daniele-athome** [comment]
This might help: #3390
